webpack.config.js

```diff
@@ -75,28 +74,6 @@ const config = {
         new webpack.DefinePlugin({
             PLUGINVERSION: JSON.stringify(packageJson.version),
         }),
-        // @ts-ignore
-        new FileManagerPlugin({
-            onStart: {
-                mkdir: [path.resolve(__dirname, 'dist/src/stepFunctions/asl/')],
-            },
-            onEnd: {
-                move: [
-                    {
-                        source: path.resolve(__dirname, 'dist/aslServer.js'),
-                        destination: path.resolve(__dirname, 'dist/src/stepFunctions/asl/aslServer.js'),
-                    },
-                    {
-                        source: path.resolve(__dirname, 'dist/aslServer.js.LICENSE.txt'),
-                        destination: path.resolve(__dirname, 'dist/src/stepFunctions/asl/aslServer.js.LICENSE.txt'),
-                    },
-                    {
-                        source: path.resolve(__dirname, 'dist/aslServer.js.map'),
-                        destination: path.resolve(__dirname, 'dist/src/stepFunctions/asl/aslServer.js.map'),
```

Comment:
@alanbo Why was this needed? `dist/src/stepFunctions/asl/aslServer.js` and `dist/src/stepFunctions/asl/aslServer.js.map` are already created by the build, just like any other `.ts` file.

Can you verify that ASL server is still working with this PR? 
You can get the artifact from this PR like so: https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#ci-artifact

Reply:
I think I can answer this question:

> Why was this needed? dist/src/stepFunctions/asl/aslServer.js and dist/src/stepFunctions/asl/aslServer.js.map are already created by the build: they are generated from src/stepFunctions/asl/aslServer.ts, just like any other .ts file.

This in only generated by `"testCompile"` before running integration tests and by `"watch"` when running the extension in debug mode.
However, when building the VSIX, nothing will be copied by webpack into `"dist/"`. The language server has to be bundled into a separate file because VSCode starts it as a separate Node process, see https://github.com/aws/aws-toolkit-vscode/blob/master/src/stepFunctions/asl/client.ts#L59

One quick way to fix it, is to change the `"entry"` section of the webpack configuration to be like this:
```
entry: {
    extension: './src/extension.ts',
    'src/stepFunctions/asl/aslServer': './src/stepFunctions/asl/aslServer.ts',
},
```
But in the long term, in my opinion it is better to move the ASL bundling in the ASL package, similar to how we did it for the SSM language server.

>Can you verify that ASL server is still working with this PR?

My guess is that the ASL server will work when running integ tests and when running the extension in debug mode. But it **won't work** when installing it from VSIX.

Reply:
> One quick way to fix it, is to change the `"entry"` section of the webpack configuration to be like this:

Ah that was done in 825fa36a5ef2679dc38ae770f047b0f . I wonder if you were looking at the previous version, which didn't have that change.

Reply:
> One quick way to fix it, is to change the `"entry"` section of the webpack configuration to be like this:

done, thanks for your help!